### PR TITLE
add malformed : the action keyword is missing test

### PR DESCRIPTION
**Pull request by jakiro2017**
propose package behaviour

### CODE REVIEW

**Comment by jakiro2017** [comment]
Are the policies allow failed unit test merge with master? I want to clear the behaviour first then implement...

**Comment by hishnash** [comment]
Sorry for the delay.

I would prefer to keep master test all passing. But feel free to build up the example tests on this branch.

File: tests/test_malformed_request.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    assert connected
+
+    await communicator.send_json_to({"pk": 2, "request_id": 1})
+
+    response = await communicator.receive_json_from()
+
+    assert response == {
+        {
+            "errors": ['Method "None" not allowed.'],
```
Comment (hishnash) [inline tests/test_malformed_request.py:38]: my be nicer for devs if we maybe mention the `action` key is missing.

**Comment by jakiro2017** [comment]
Hi sorry for the delay too, due to my personal matter, I will delay it a bit, but sure will do

**Comment by hishnash** [comment]
this code has moved on if you feel this is still needed feel free to open a new PR

**Pull request closed without merging**